### DISCUSSION TITLE
Select icon for variant, genotype, samples

### DISCUSSION THREAD

**Discussion by dridk** [labsquare/CuteVariant-python#446] [Ideas]
It would be greate to distinguish variant, samples, genotypes classification. 

Help us to select and vote for icons : 
https://pictogrammers.github.io/@mdi/font/5.4.55/

**Comment by dridk** [comment]
I would suggest primitive shape : 

- circle ( variant ) 
- square ( samples ) 
- losange ( genotype )

**Comment by antonylebechec** [comment]
I suggest:
- Samples
   -  default: <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169369651-846e0ff2-c126-48d1-ae51-448346d7d481.png"> 0xF0B55 or <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169371558-15986708-0fae-4889-9c2d-0de40e4d896d.png"> F0004
   - favorite: <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169369843-852d25b2-00e4-4245-b026-880e21faa975.png"> 0xF0009 or <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169371603-1c635bce-fa64-48d4-9fba-c56df4c64405.png"> F0008
- Variants
   - default: <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169371952-eb305539-e6b8-4893-8ead-8abb529983ea.png"> 0xF0130 or <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169371952-eb305539-e6b8-4893-8ead-8abb529983ea.png"> 0xF0130 or <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169371952-eb305539-e6b8-4893-8ead-8abb529983ea.png"> 0xF0130
   - favorite: <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169372143-ffc4062b-3548-4f6e-80a5-e5a30da9d593.png"> 0xF0133 or <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169372065-956543f1-8aa5-4fe8-a066-0ced3bfbb375.png"> 0XF012F or <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169372238-48f0e8a6-84ab-4bfe-bba9-4bccfa70b240.png"> 0xF0134
- Genotypes
   - None (-1): <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169372506-d1f0a457-34f0-4824-abb9-c79b1a9d0697.png"> 0xF10D3
   - Wild (0): <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169372608-cd1d445a-0f56-4ef7-9ccb-f47bb8984466.png"> 0xF0766
   - Het (1): <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169372708-ce55e371-926a-44af-b844-6492c96bfd14.png"> 0xF0AA1
   - Hom (2): <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169372884-d028a4ea-d940-4e0a-b97b-073744011312.png"> 0xF0AA5
   - unknown (3+): <img width="27" alt="image" src="https://user-images.githubusercontent.com/40463532/169373052-e3a12ba1-3390-4a8c-95c8-97a2c3de6c4c.png"> 0xF0B38

colors: related to classification
